lab_2_keywords_cooccurrence/main.py

```diff
@@ -84,7 +154,17 @@ def calculate_cumulative_score_for_candidates(candidate_keyword_phrases: KeyPhra
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not (check_list(candidate_keyword_phrases, tuple, False) and check_dict(word_scores, str, float, False)):
+        return None
+    cumulative_score = {}
+    for phrase in candidate_keyword_phrases:
+        value = 0
+        for word in phrase:
+            if word not in word_scores.keys():
+                return None
+            value += int(word_scores[word])
```

Comment:
another conversion to int 

if it is indeed unavoidable, it is not your fault
however, it does contradict both the task and the type annotations